pyupgrade.py

```diff
@@ -93,8 +93,9 @@ def _convert_tup(tup):  # type: (DotFormatPart) -> str
     return j.join(_convert_tup(tup) for tup in parsed)
 
 
-# if py3+ use tokenize.cookie_re
+# if py3+ use tokenize.cookie_re and tokenize.blank_re
 cookie_re = re.compile('^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)')
+blank_re = re.compile(r'^[ \t\f]*(?:[#\r\n]|$)', re.ASCII)
```

Comment:
I don't know if the `re.ASCII` part makes sense, I just copy pasted it from `tokenize`.